packages/kit/src/core/tsconfig.js

```diff
@@ -29,9 +29,9 @@ export function generate_tsconfig(config) {
 	/** @type {string[]} */
 	const include = [];
 	dirs.forEach((dir) => {
-		include.push(config_relative(`${dir}/**/*.js`));
-		include.push(config_relative(`${dir}/**/*.ts`));
-		include.push(config_relative(`${dir}/**/*.svelte`));
+		include.push(config_relative(`./${dir}/**/*.js`));
+		include.push(config_relative(`./${dir}/**/*.ts`));
+		include.push(config_relative(`./${dir}/**/*.svelte`));
```

Comment:
why do we need this?

Reply:
Probably the same reasoning as for adding the `./` prefix in the other places

Reply:
```suggestion
		include.push(config_relative(`${dir}/**/*.js`));
		include.push(config_relative(`${dir}/**/*.ts`));
		include.push(config_relative(`${dir}/**/*.svelte`));
```

Reply:
removed